packages/template-retail-react-app/app/components/passwordless-login/index.jsx

```diff
@@ -12,20 +12,17 @@ import {Button, Divider, Stack, Text} from '@salesforce/retail-react-app/app/com
 import LoginFields from '@salesforce/retail-react-app/app/components/forms/login-fields'
 import StandardLogin from '@salesforce/retail-react-app/app/components/standard-login'
 import SocialLogin from '@salesforce/retail-react-app/app/components/social-login'
-import {LOGIN_TYPES} from '@salesforce/retail-react-app/app/constants'
```

Comment:
Should we delete this constant from `constants.js` if we are not using it anymore?